webpack/scenes/Subscriptions/Details/SubscriptionDetailEnabledProducts.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { notify } from '../../../move_to_foreman/foreman_toast_notifications';
+
+class SubscriptionDetailEnabledProducts extends Component {
```

Comment:
When no react lifecycle or the component state is involved, it better to use [stateless component](https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc).